python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/aaa-authorization.j2

```diff
@@ -17,12 +14,12 @@ aaa authorization dynamic dot1x additional-groups group {{ aaa_authorization.dyn
 {%     if aaa_authorization.exec.default is arista.avd.defined %}
 aaa authorization exec default {{ aaa_authorization.exec.default }}
 {%     endif %}
-{%     if aaa_authorization.config_commands is arista.avd.defined(false) %}
-no aaa authorization config-commands
-{%     endif %}
 {%     if aaa_authorization.commands.all_default is arista.avd.defined %}
 aaa authorization commands all default {{ aaa_authorization.commands.all_default }}
 {%     endif %}
+{%     if aaa_authorization.config_commands is arista.avd.defined(false) %}
+no aaa authorization config-commands
```

Comment:
is this one not tested (not seeing any change in ordering in the molecule) or not tested in conjunction with another CLI entry that would show the ordering change?